proxy_internal.c

```diff
@@ -1204,6 +1216,16 @@ static void process_mset_cmd(LIBEVENT_THREAD *t, mcp_parser_t *pr, mc_resp *resp
                 META_CHAR(p, 'c');
                 p = itoa_u64(cas, p);
                 break;
+            case 's':
```

Comment:
if the `ms` is supposed to return say `NS` due to cas conflict or something then will `s` flag still return the old item size?

Reply:
okay - then it would be just safe to ignore! thanks!